### PR TITLE
Replace Contact Summary Events tab with Admin UI Search Display and Form

### PR DESCRIPTION
**Pull request by guyiac**
Overview
----------------------------------------
Replace Contact Summary Events tab with Admin UI Search Display and Form per Coleman's request

Before
----------------------------------------
Standard Events Tab

After
----------------------------------------
Events tab of Contact Summary is a Search Display in a FormBuilder Form

Technical Details
----------------------------------------
Pretty straightforward

Comments
----------------------------------------
First commit ever. Be kind.

### CODE REVIEW

**Comment by seamuslee001** [comment]
Jenkins add to whitelist

**Comment by kcristiano** [comment]
jenkins test this please

File: ext/civicrm_admin_ui/ang/afsearchEvents.aff.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<div af-fieldset="">
+  <div class="af-markup">
+    <div class="help">
+    <p>This page lists all event registrations for this contact since inception. Click Add Event Registration below to register this contact for an event. Click Submit Credit Card Event Registration to process a new registration on behalf of the participant using their credit card.</p>
+    </div>
```
Comment (colemanw) [inline ext/civicrm_admin_ui/ang/afsearchEvents.aff.html:5]: We need to add `ts()` around this test. I started doing that, and that got me thinking about the text itself and how it's a big mouthful of nothing. It just explains a self-explanatory table, then tells you you can click on some self-explanatory buttons and that they will do the thing they already say they will do.

**Comment by colemanw** [comment]
Ok I've reviewed, tested & cleaned this up, using a linksProvider callback to give the multiple register + credit card buttons.
The one thing I removed is the self-service "Transfer or Cancel" link which seems more appropriate to the front-end than the backend. I see it was added in #10695 but I still don't understand the use-case for it. It's still accessible by clicking the "View" link and then the self-service button is there... maybe that's good enough for whoever wanted it added?

**Comment by guyiac** [comment]
> Ok I've reviewed, tested & cleaned this up, using a linksProvider callback to give the multiple register + credit card buttons. The one thing I removed is the self-service "Transfer or Cancel" link which seems more appropriate to the front-end than the backend. I see it was added in #10695 but I still don't understand the use-case for it. It's still accessible by clicking the "View" link and then the self-service button is there... maybe that's good enough for whoever wanted it added?

Isn't there a use case for an admin transferring or cancelling a registration if someone calls or shows up at the front desk?

**Comment by eileenmcnaughton** [comment]
@colemanw this has passed & is MOP but just want to check that you have seen @guyiac's comment

**Comment by colemanw** [comment]
@guyiac @eileenmcnaughton ok I have added the self-serve link back in.

**Comment by francescbassas** [comment]
This introduces a little regression. Someone can review it? https://github.com/civicrm/civicrm-core/pull/30086

**Pull request merged**